src/vicsetup/libvicsetup/Makefile

```diff
@@ -0,0 +1,16 @@
+include ../defs.mak
+
+ARCHIVE = libvicsetup.a
+
+CFLAGS += -Wno-type-limits
+
+INCLUDES += -I/usr/local/include
+INCLUDES += -I../../../third_party/argon2/include
```

Comment:
Don't hard-code this here. vicsetup is an independent library that should reach out. It can receive the path to argon2 via some make variable that's passed in.

Reply:
Added **LIBJSON_DIR** and **ARGON2_DIR** make variables and check whether these directories exist.